http-client/src/main/java/io/micronaut/http/client/netty/ConnectionManager.java

```diff
@@ -193,6 +193,8 @@ public class ConnectionManager {
         this.clientCustomizer = from.clientCustomizer;
         this.informationalServiceId = from.informationalServiceId;
         this.nettyClientSslBuilder = from.nettyClientSslBuilder;
+
+        refresh();
```

Comment:
I don't think this is necessary

Reply:
This is necessary cause it constructor is used for copying ConnectionManager to new instance so we need to bootstrap new instance to setup states. For example AtomicBoolean running is false by default and without refresh() isRunning will return false. I add this line cause i had failed tests. Hope you will get me, thanks for your review.